docs/faq.md

```diff
@@ -292,9 +292,38 @@ called *.NET Standard*.
 
 ## Can you explain the assemblies and type forwarding in more detail?
 
-Yes. For now, take a look at the [.NET Standard 2.0 spec][netstandard-spec], as
+Yes. I made video explaining how .NET Standard works [under the
+covers](https://www.youtube.com/watch?v=vg6nR7hS2lI&list=PLRAdsfhKI4OWx321A_pr-7HhRNk7wOLLY&index=4).
+You can also take a look at the [.NET Standard 2.0 spec][netstandard-spec], as
 it has some diagrams.
 
+## Should I reference the meta package or should I reference individual packages?
+
+In the past, we've given developers the recommendation to not reference the meta
+package (`NETStandard.Library`) from NuGet packages but instead reference
+individual packages, like `System.Runtime` and `System.Collections`. The
+rationale was that we thought of the meta package as a shorthand for a bunch of
+packages that were the actual atomic building blocks of the .NET platform. The
+assumption was: we might end up creating another .NET platform that only
+supports some these atomic blocks bot not all of them. There were also concerns
+regarding how our tooling deals with large package graphs.
+
+Moving forward, we'll simplify this:
+
+1. **.NET Standard is an atomic building block**. In other words, new platforms
+aren't allowed to subset .NET Standard -- they have to implement all of it.
+
+2. **We're moving away from using packages to describe our platforms**,
+including .NET Standard.
+
+This means, you'll not have to reference any NuGet packages for .NET Standard
+anymore. You expressed your dependency with the lib folder, which is exactly how
+it has worked for all other .NET platforms, in particular .NET Framework.
+
+However, right now our tooling will still burn in the reference to
```

Comment:
> do we still think it is best for folks to reference NETStandard.Library meta-package?

Right now that's the expectation.  There are a number of those older platforms that need out of band facades/libs and omitting a reference just means that 90% of the time the app developer needs to add it himself.  I think the reason why NS2.0 is different is because it supports a smaller number of frameworks today and the new goal is to have it be a representation of what is supported by default in the framework (with desktop being the one exception).

> Also do we still plan to update it to point at the earlier versions of the packages when targeting a netstandard TFM?

This issue is tracking that: https://github.com/dotnet/corefx/issues/15735